src/runtime_src/core/tools/common/ReportPlatforms.cpp

```diff
@@ -23,6 +23,7 @@
 
 // 3rd Party Library - Include Files
 #include <boost/property_tree/json_parser.hpp>
+#include <boost/optional/optional.hpp>

```

Comment:
Please make sure that the #includes are in alphabetical order.   
For short lists like this one, it doesn't matter, for for longer ones, it helps quickly recognize and remove duplicates.